helper/plugin/grpc_provider.go

```diff
@@ -882,6 +890,16 @@ func (s *GRPCProviderServer) ApplyResourceChange(_ context.Context, req *proto.A
 		}
 	}
 
+	pmSchemaBlock := s.getProviderMetaSchemaBlock()
```

Comment:
Is there a reason this is only implemented for `ApplyResourceChange`

Reply:
I think because I'm a little sketchy on the relation of GRPCProvider to GRPCProviderServer... I'll look into it.